ftp/1.5/modules/ROOT/pages/ftp-write.adoc

```diff
@@ -34,15 +71,39 @@ The next example uses the FTP connector to write the content:
 
 == Write into a Directory
 
-If any of the `a`, `b`, or `c` directories in the example below do not exist, the Write operation  fails by default. However, if you set the `createParentDirectories` to `true`, the connector  automatically creates any missing directories.
+The *Write* operation fails by default if you set nonexistent directories in *Path*. Set *Create parent directories* to *True (Default)* to automatically create any missing directories.
+
+In the following example, if any of the `a`, `b`, or `c` directories set in *Path* do not exist, the connector creates them.
+
+image::ftp-write-directories.png[Write into a directory General settings in Studio]
+
+In the XML editor, the configuration looks like this:
 
-.Example: FTP Connector writes to a Directory
 [source,xml]
 ----
 <ftp:write path="a/b/c/myFile.txt" />
 ----
 
-// INCLUDE FOR THESE SECTIONS:
-// == Writing to Existing Files
-// == Locks
-include::partial$common-write.adoc[]
```

Comment:
This .adoc file uses _**includes**_ to reuse the same content in different .adoc files. In this case the sections **_Write to Existing Files and Locks_** are defined in the _**common-write.adoc**_ file which is located in the **_partials_** folder of the FTP Connector.
When you have time review the Wiki guidelines on partials to understand how they work https://confluence.internal.salesforce.com/display/MTDT/Reuse+Content+in+a+Single+Repo+with+_partials